mock/py/mock.py

```diff
@@ -381,17 +375,26 @@ def command_parse():
 
     # Bootstrap options
     parser.add_option('--bootstrap-chroot', dest='bootstrapchroot', action='store_true',
-                      help="build in two stages, using chroot rpm for creating the build chroot")
+                      help="build in two stages, using chroot rpm for creating the build chroot",
+                      default=None)
     parser.add_option('--no-bootstrap-chroot', dest='bootstrapchroot', action='store_false',
-                      help="build in a single stage, using system rpm for creating the build chroot")
+                      help="build in a single stage, using system rpm for creating the build chroot",
+                      default=None)
 
     parser.add_option('--use-bootstrap-image', dest='usebootstrapimage', action='store_true',
                       help="create bootstrap chroot from container image (turns "
-                           "--bootstrap-chroot on)")
+                           "--bootstrap-chroot on)", default=None)
     parser.add_option('--no-bootstrap-image', dest='usebootstrapimage', action='store_false',
-                      help="don't create bootstrap chroot from container image")
+                      help="don't create bootstrap chroot from container image", default=None)
+
+    (options, args) = parser.parse_known_args()
```

Comment:
why the use of _known_args? why not parse_args?

Reply:
Because we want to retrun "tuple", with `options` + leftovers in `args` (to mimic previous behavior).
`parse_args` returns just `options`.